### PR TITLE
Updated flycast and libretro flycast

### PR DESCRIPTION
**Pull request by bladeoner**
- Fix retro achievements
- Fix flycast exit

### CODE REVIEW

File: package/batocera/emulators/flycast/atomiswave.flycast.keys
Comment (acmeplus) [inline package/batocera/emulators/flycast/atomiswave.flycast.keys:1]: Do we really need this change? killall should work without the full path. If it's not needed I suggest to leave it as it was, same for the other .keys file below

Comment (acmeplus) [reply package/batocera/emulators/flycast/atomiswave.flycast.keys:1]: @bladeoner check that and if it's not needed revert those .keys files. Other than that it looks good.

Comment (bladeoner) [reply package/batocera/emulators/flycast/atomiswave.flycast.keys:1]: I will revert the keys files they are not necessary I agree, the other changes are more important.

Comment (bladeoner) [reply package/batocera/emulators/flycast/atomiswave.flycast.keys:1]: @acmeplus removed the changes to the keys and rebased.

**Pull request merged**